sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -603,6 +627,36 @@ private boolean getSuccess(SpanData span) {
 
     @Nullable
     public static String getHttpUrlFromServerSpan(Attributes attributes) {
+        String httpUrl = getHttpUrlFromServerSpanStableSemconv(attributes);
+        if (httpUrl != null) {
+            return httpUrl;
+        }
+        return getHttpUrlFromServerSpanOldSemconv(attributes);
+    }
+
+    @Nullable
+    private static String getHttpUrlFromServerSpanStableSemconv(Attributes attributes) {
```

Comment:
can you please add the link to the otel spec for this?

Reply:
I don't think there's an otel spec for how to build http url from its components, this might be the most relevent spec: https://www.rfc-editor.org/rfc/rfc2616